weave-js/src/components/Panel2/ChildPanelExportReport/ReportSelection.tsx

```diff
@@ -106,50 +146,72 @@ export const ReportSelection = ({
         className="mb-4 block font-semibold text-moon-800">
         Destination report
       </label>
-      {selectedReport == null && (
-        <Select<ReportOption, false>
-          className="mb-16"
-          id="report selector"
-          isLoading={reports.loading}
-          isDisabled={
-            entities.loading || reports.loading || reports.result.length === 0
-          }
-          options={reports.result ?? []}
-          placeholder={
-            !reports.loading && reports.result.length === 0
-              ? 'No reports found.'
-              : 'Select a report...'
-          }
-          getOptionLabel={option => option.name}
-          getOptionValue={option => option.id ?? ''}
-          value={selectedReport}
-          onChange={selected => {
-            if (selected != null) {
-              setSelectedReport(selected);
-            }
-          }}
-          components={customReportSelectComps}
-          isSearchable
+      {selectedReport != null && selectedReport.name !== NEW_REPORT_OPTION && (
+        <SelectedExistingReport
```

Comment:
**thought:** I think it's a bit odd that this piece has been pulled outside the context of the Select. I feel like it would be more natural to fold this into the `SingleValue` custom select comp?

Reply:
Okay okay so I attempted to stick this into the `SingleValue` custom comp and tried `SelectContainer` since we're trying to replace the entire selector with this `SelectedExistingReport`. 

At the end, I still need to replace the `Select` with this custom `SelectedExistedReport` because we're restricted by the functions `SelectContainer` props provides us. We get more flexibility if we have this piece of logic outside of Select. 

SingleValue doesn't seem to be what we want based on [docs](https://react-select.com/components)
![Screenshot 2023-09-19 at 2 24 19 PM](https://github.com/wandb/weave/assets/103448723/720b45ab-6abf-476d-a2e9-99683648c5c1)
![Screenshot 2023-09-19 at 2 00 07 PM](https://github.com/wandb/weave/assets/103448723/b33e257a-e278-4ec9-83a5-e2803e9a2663)

SelectContainer makes it difficult to `setSelectedReport(null)` when you click the `X` button. 


![Screenshot 2023-09-19 at 2 22 35 PM](https://github.com/wandb/weave/assets/103448723/5edaf695-c496-42e5-af30-f746e64a22ab)

Reply:
Ok yeah don't worry about forcing our design needs onto react-select, i think there's better places to put our time